build/android/native/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.library'
+import org.ajoberstar.grgit.Grgit
```

Comment:
When I understand it right, what's about git submodules, to get rid of this ?

Reply:
Answer above